src/BusinessLayer/CommunicationService.cpp

```diff
@@ -28,21 +29,20 @@ void CommunicationService::setPeripheralSerialPort()
     outputPeripheral_ = new SerialPortPeripheral(*outputDevice_);
 }
 
+void CommunicationService::setPeripheralInternet()
+{
+    outputDevice_ = new QSerialPort();
```

Comment:
You can't just rename one thing if you're starting to do other stuff. Keep in mind the big picture. Right now it looks like the internet stuff got added. After you're done working with any code base, it should look like it's always been like that.